firebase-ml-modeldownloader/api.txt

```diff
@@ -61,7 +61,7 @@ package com.google.firebase.ml.modeldownloader {
     method @NonNull public com.google.android.gms.tasks.Task<com.google.firebase.ml.modeldownloader.CustomModel> getModel(@NonNull String, @NonNull com.google.firebase.ml.modeldownloader.DownloadType, @Nullable com.google.firebase.ml.modeldownloader.CustomModelDownloadConditions);
     method public long getModelDownloadId(@NonNull String);
     method @NonNull public com.google.android.gms.tasks.Task<java.util.Set<com.google.firebase.ml.modeldownloader.CustomModel>> listDownloadedModels();
-    method public void setStatsCollectionEnabled(boolean);
+    method public void setDataCollectionMlDownloaderState(boolean);
```

Comment:
I think the approved naming pattern is `set${product}CollectionEnabled(Boolean)`

cc @VinayGuthal to confirm

Reply:
Looking through - your recommendation is the format used by crashlytics and performance - so updated to match.

Reply:
@vkryachko is right.

Reply:
Done - updated method to setModelDownloaderCollectionEnabled(Boolean) and added null edges to reset to default firebase level data collection.